CHANGELOG.rst

```diff
@@ -1,6 +1,59 @@
 Changelog
 =========
 
+0.8.0 (2018-11-08)
+-----------------
+
+MLflow 0.8.0 introduces several major features:
+
+- Dramatically improved UI for comparing experiment run results:
+
+  - Metrics and parameters are by default grouped into a single column, to avoid an explosion of mostly-empty columns. Individual metrics and parameters can be moved into their own column to help compare across rows.
+  - Runs which are "nested" inside other runs (e.g., as part of a hyperparameter search or multistep workflow) now show up grouped by their parent run, and can be expanded or collapsed altogether.
+  - Run names (as opposed to automatically generated run UUIDs) now show up instead of the run id, making comparing runs in graphs easier.
+  - The state of the run results table, including filters, sorting, and expanded rows, is persisted in browser local storage, making it easier to go back and forth between an individual run view and the table.
+
+- Support for deploying models as Docker containers directly to Azure Machine Learning Studio Workspace. This joins our SageMaker support to make it easy to deploy models saved using MLflow to the cloud.
```

Comment:
Since we had Azure ML support before, we should probably not say "this joins our SageMaker support"; this is just a new version of the Azure ML product basically.